### ISSUE TITLE
Support for an All option in template query

### ISSUE THREAD

**Issue by pcerny** [jakubplichta/grafana-dashboard-builder#95]
Grafana has an All option to add All value to query variable value.
graphite value: `site.cluster.node.prefix-value-suffix.subvalue`
Example setup for prefix-value-suffix:
* variable name - `var1`
* values query - `$site.$cluster.$node.prefix-*`
* All option - has several flavors glob, wildcard, regex wildcard, regex values

```json
"type": "query",
"datasource": null,
"refresh_on_load": false,
"name": "var1",
"options": [
  {
    "text": "All",
    "value": "\*"
  }
],
"includeAll": true,
"allFormat": "wildcard",
"query": "$site.$cluster.$node.prefix-*",
```

**Comment by jakubplichta** [comment]
fixed by #96

**Comment by jakubplichta** [comment]
adding support for configurable `allFormat` and `allValue`
